docs/source/development-guide/docker.rst

```diff
@@ -0,0 +1,37 @@
+Docker Workflow
+----------------
+
+This page describes how to build and run a Docker Image Locally and in AWS.
+
+Building and Running a Docker Image Locally
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To build the image run the following command from the directory containing the Dockerfile. You might add -t option to tag your image
+and --rm to remove intermediate containers after the build is done.
+
+    `docker build -t <image name> --rm .`
+
+Now we can run our image.
+
+    `docker run --rm -it --volume="$(pwd)/imap_processing/efs:/mnt/spice" <image name> --instrument <instrument> --level <level>`
+
+Building and running a docker image in AWS
+"""""""""""""""""""""""""""""""""""""""""""
+
+Make certain to set the correct AWS Profile.
+
+    `export AWS_PROFILE=<your profile>`
+
+Login to the ECR.
+
+    `aws ecr get-login-password --region <region> | docker login --username AWS --password-stdin <ECR URI>`
+
+Build the Docker image.
+
+    `docker build -t <image name> .`
+
+Tag the image and push to the ECR.
+
+    `docker tag <tag> <ECR URI>`
+
+    `docker push <ECR URI>`
```

Comment:
Is this the place where we would set image tag with version?

Reply:
We can assign the tag name during the build :)